driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/access.bpf.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2022 The Falco Authors.
+ *
+ * This file is dual licensed under either the MIT or GPL 2. See MIT.txt
+ * or GPL2.txt for full copies of the license.
+ */
+
+#include <helpers/interfaces/fixed_size_event.h>
+#include <helpers/interfaces/variable_size_event.h>
+
+/*=============================== ENTER EVENT ===========================*/
+
+SEC("tp_btf/sys_enter")
+int BPF_PROG(access_e,
+	     struct pt_regs *regs,
+	     long id)
+{
+	struct auxiliary_map *auxmap = auxmap__get();
+	if(!auxmap)
+	{
+		return 0;
+	}
+
+	auxmap__preload_event_header(auxmap, PPME_SYSCALL_ACCESS_E);
+
+	/*=============================== COLLECT PARAMETERS  ===========================*/
+
+	/* Parameter 1: mode (type: PT_UINT32) */
+	unsigned long mode = (u32)extract__syscall_argument(regs, 1);
+	auxmap__store_u32_param(auxmap, (u32)access_flags_to_scap(mode));
+
+	/*=============================== COLLECT PARAMETERS  ===========================*/
+
+	auxmap__submit_event(auxmap);
+
+	return 0;
+}
+
+/*=============================== ENTER EVENT ===========================*/
+
+
+/*=============================== EXIT EVENT ===========================*/
+
+SEC("tp_btf/sys_exit")
+int BPF_PROG(access_x,
+	     struct pt_regs *regs,
+	     long ret)
+{
+	struct auxiliary_map *auxmap = auxmap__get();
+	if(!auxmap)
+	{
+		return 0;
+	}
+
+	auxmap__preload_event_header(auxmap, PPME_SYSCALL_ACCESS_X);
+
+	/*=============================== COLLECT PARAMETERS  ===========================*/
+
+        /* Parameter 1: res (type: PT_ERRNO) */
+        auxmap__store_s64_param(auxmap, ret);
+
+	/* Parameter 2: pathname (type: PT_FSPATH) */
+	unsigned long mode = (u32)extract__syscall_argument(regs, 1);
+	auxmap__store_u32_param(auxmap, (u32)access_flags_to_scap(mode));
+
+	/*=============================== COLLECT PARAMETERS  ===========================*/
+
```

Comment:
The event finalization is missing here which causes the a test case failure:
````
[==========] Running 2 tests from 2 test suites.
[----------] Global test environment set-up.
[----------] 1 test from SyscallExit
[ RUN      ] SyscallExit.accessX
/root/git/falcosecurity/libs/test/modern_bpf/event_class/event_class.cpp:427: Failure
Expected equality of these values:
  m_event_header->len
    Which is: 0
  m_event_len
    Which is: 61
'event_len' in the header is not correct.

[  FAILED  ] SyscallExit.accessX (0 ms)
````
Adding the following:
````suggestion

       auxmap__finalize_event_header(auxmap);

````
let the tests passes (tried on `s390x`):
````
Note: Google Test filter = *access*
[==========] Running 2 tests from 2 test suites.
[----------] Global test environment set-up.
[----------] 1 test from SyscallExit
[ RUN      ] SyscallExit.accessX
[       OK ] SyscallExit.accessX (0 ms)
[----------] 1 test from SyscallExit (0 ms total)

[----------] 1 test from SyscallEnter
[ RUN      ] SyscallEnter.accessE
[       OK ] SyscallEnter.accessE (39 ms)
[----------] 1 test from SyscallEnter (39 ms total)

[----------] Global test environment tear-down
[==========] 2 tests from 2 test suites ran. (39 ms total)
[  PASSED  ] 2 tests.
````